include/x86.h

```diff
@@ -203,6 +203,19 @@ typedef struct cs_x86_op {
 		bool avx_zero_opmask;
 } cs_x86_op;
 
+typedef struct cs_x86_encoding {
+	// ModR/M offset, or 0 when irrelevant
+	//uint8_t modrm_offset;
```

Comment:
will we remove these commented lines above?

Reply:
yea probably best to do so for now, i dont use modrm and if someone wants it later then can add it again from encoding. Or maybe i'll do it later when i merge to next